src/app/components/ThemeProvider/fontFacesLazy.ts

```diff
@@ -24,7 +24,7 @@ const NOTO_SERIF_BENGALI_FONTS_DIR =
   'https://ws-downloads.files.bbci.co.uk/fonts/NotoSerifBengali/v1.00/';
 
 const REITH_QALAM_FONTS_DIR =
-  'https://ws-downloads.files.bbci.co.uk/fonts/ReithQalam/v1.310/';
+  'https://static.files.bbci.co.uk/fonts/ReithQalam/v1.310/';
```

Comment:
According to https://paper.dropbox.com/doc/RFC-Move-WS-Fonts-to-Simorgh--CqLoh3rvTXsahM48iZtzTw0vAg-votGTRUOGTAkPwXjG8pF1#:uid=875214826650728140482684&h2=Could-we-move-the-Font-files? this needs to be
```suggestion
  'https://static.files.bbci.co.uk/fonts/reith-qalam/v1.310/';
```

Reply:
And then below - where the `REITH_QALAM_FONTS_DIR` is referenced, we also need to update the names of the normal & bold fonts (as normal.woff & bold.woff do not exist, but https://static.files.bbci.co.uk/fonts/reith-qalam/1.310/BBCReithQalam_W_Rg.woff and https://static.files.bbci.co.uk/fonts/reith-qalam/1.310/BBCReithQalam_W_Bd.woff do)